pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -89,6 +117,33 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 			return skipReason, err
 		}
 
+		// if PVC not found in cache, re-check from apiserver directly to make sure the PVC really not exist
+		_, err = opc.kubeCli.CoreV1().PersistentVolumeClaims(ns).Get(pvcName, metav1.GetOptions{})
+		if err == nil {
+			skipReason[podName] = skipReasonOrphanPodsCleanerPVCIsFound
+			continue
+		}
+		if !errors.IsNotFound(err) {
+			return skipReason, err
+		}
+
+		// if the PVC is not found in apiserver (also informer cache) and the
+		// phase of the Pod is Pending, delete it and let the stateful
+		// controller to create the pod and its PVC(s) again
+		apiPod, err := opc.kubeCli.CoreV1().Pods(pod.Namespace).Get(pod.Name, metav1.GetOptions{})
```

Comment:
Use `ns` and `podName` instead of `pod.Namespace` and `pod.Name`

Reply:
fixed